.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+(Title field)
```

Comment:
Do we really need the extra sub-folder? like wouldn't `.github/pull_request_template.md` be enough structure?

Reply:
You're right, not necessary in scaffolding